Userland/Libraries/LibGUI/TextDocument.cpp

```diff
@@ -786,6 +786,10 @@ void InsertTextCommand::perform_formatting(const TextDocument::Client& client)
 
     for (auto input_char : m_text) {
         if (input_char == '\n') {
+            size_t spaces_at_end = 0;
+            if (dest_line.length() == line_indentation and !at_start_of_line)
```

Comment:
Please use `&&` instead of `and`

Reply:
Sorry about that! Didn't even know and was a keyword in C++, I must have brought it from Python...
Anyway, I changed it.